pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -607,10 +611,32 @@ message ExecRequest {
 }
 
 message ExecResponse {
-    // Streaming stdout
-    optional bytes stdout = 1;
-    // Streaming stderr
-    optional bytes stderr = 2;
+    // The fully qualified URL of the exec streaming server
+    optional string url = 1;
+}
+
+message AttachRequest {
+    // The id of the container
+    optional string container_id = 1;
+    // Streaming stdin
+    optional bytes stdin = 2;
```

Comment:
That's reasonable, but `bytes` indicates a streaming behavior while this is not a streaming API.

Do you think changing `stdin` to `bool` is enough?

And I think  `bool tty` should also be added to `AttachRequest` and `ExecRequest`.